examples/sinatra/spec/app_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'POST /saml-login ' do
+  it 'encodes a SAMLResponse' do
+    allow_any_instance_of(SamlIdp::Controller).to receive(:decode_request).and_return(true)
+    allow_any_instance_of(SamlIdp::Controller).to receive(:encode_response).and_return("SAMLResponse")
+
+    post("/saml-login")
+
+    expect(last_response).to be_ok
+  end
+end
```

Comment:
here's a simple test @Zogoo - i dont want to test the gem's internals, but this seems to be to be a pointless test.

happy to iterate to your liking but im not sure what you are hoping to accomplish here? I don't think it makes sense to use a real SAMLRequest (that would be testing the library).

Reply:
I was expecting more working example that proof this Sinatra example work with our future changes.
I'm afraid that if we stub responses and if we merge some pull request that breaks your "sample" then your sample no longer gets useful for others.
So, could you please add one proper response test for your sample app?